src/test_rpc.ts

```diff
@@ -130,6 +130,37 @@ export default async function test(client: TestClient) {
     await new TestCase(schema, handler, test).run();
   });
 
+  it('handles multiple parameters', async () => {
+    const schema = `
+export interface Test {
+  bar: {
+    params: {
+      b: string;
+      a: number;
+      c: string | undefined;
```

Comment:
please test optional params (`c?: string`), this case already works without the fix